src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain other methods directly after it.